src/database/migrations/20240813123014-create-activities-table.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('activities', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
+			user_id: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			action_id: {
+				allowNull: false,
+				type: Sequelize.INTEGER,
+			},
+			object_id: {
+				allowNull: false,
+				primaryKey: true,
+				type: Sequelize.STRING,
+			},
+			object_type: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			organization_id: {
+				allowNull: false,
+				type: Sequelize.STRING,
+				primaryKey: true,
+			},
+			created_at: {
+				allowNull: false,
+				type: Sequelize.DATE,
+			},
+			updated_at: {
+				allowNull: false,
+				type: Sequelize.DATE,
+			},
+			deleted_at: {
+				type: Sequelize.DATE,
+			},
+		})
+		//add index
```

Comment:
@priyanka-TL why do we need orgid and user id as index? just object ID is enough that is the only thing will be queries with.

Reply:
@aks30 previously i have designed like org admin can see all users actions, but as per the suggestion i have removed that. 
hence removing the primary key also